### DISCUSSION TITLE
JuMP-dev 2024

### DISCUSSION THREAD

**Discussion by franckgaga** [franckgaga/ModelPredictiveControl.jl#53] [General]

**Comment by franckgaga** [comment]
@odow I'm starting the presentation, what's the time limit in the talk ?

**Reply by odow** [reply]
15 min slots, so 10-12 minutes + 3 minutes questions.

**Comment by franckgaga** [comment]
alright thanks!

**Comment by franckgaga** [comment]
@odow do you know if we will be able to connect our own laptop over there ?

**Reply by odow** [reply]
Yes, you should bring and use your own laptop to present.

**Reply by odow** [reply]
I guess I was wrong 😆 you need to use the built-in computer. Miles just sent an email.

**Reply by franckgaga** [reply]
Yes, I just saw that XD. It does not matter much, I wanted to test the controller gadget in KDE Connect. Oh well :disappointed:

**Reply by franckgaga** [reply]
@blegat Est-ce que tu peux m'envoyer le pluto workbook de ta présentation de samedi dernier ? Merci!

**Reply by blegat** [reply]
Pour le moment, il a besoin d'utiliser des branches de pas mal de packages donc ça serait difficile à reproduire. Je le mettrai à jour au fur et à mesure ici: https://github.com/blegat/SumOfSquaresSlides/tree/bl/2024_JuMP
